### PR TITLE
Unique food

### LINKED ISSUE #24: Require food names to be unique
Add a validation that food names are unique

### PR DESCRIPTION
**Pull request by ryanmillergm**
**Change food name requirement to unique:**
 - Adds unique: true to food migration
 - Adds food name must be unique to test
 
Resolves #24 
 
**The following checks have been completed:**
 - [x] Tested new feature(s) as well as any feasible edge cases
 - [ ] Updated README for changes (new endpoints, new packages, etc)
 - [x] Merged in the latest master & resolved all merge conflicts
 - [x] Ran database migrations
 - [x] Ran the test suite - all tests are passing (or maybe skipped)
 - [x] Checked affected endpoints in Postman

### CODE REVIEW

File: test/requests/foods/create.test.js [hunk 1]
```diff
@@ -61,5 +61,30 @@ var specHelper = require('../../specHelper');
...
+              request(app)
+                .post('/api/v1/foods')
+                  .send({
+                    food: {
+                      name: "Mint",
+                      calories: 14
+                    }
+                  })
+                  .then(response => {
+                    expect(response.body.error.errors[0].message).to.equal("name must be unique")
```
Comment (chakeresa) [inline test/requests/foods/create.test.js]: Let's add an assertion for a particular status code (probably 422) and make sure the controller catches it appropriately. Also need to call out in the README that food names must be unique and state what response you'll get if you try do create a duplicate.

File: migrations/20190820204114-create-food.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       name: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
+        unique: true
```
Comment (chakeresa) [inline migrations/20190820204114-create-food.js]: Need to revert this file back to the original -- never edit a migration file that's already been run on a computer other than your own (especially if it's already run in production).

**Comment by chakeresa** [comment]
Let's just start a new branch with a new migration file instead of editing the existing one.
